### PR TITLE
Updated text on Code of Ethics page

### PR DESCRIPTION
**Pull request by zoebentley**
Changes: 
-Updated page text to match this document: https://docs.google.com/document/d/18o88J_C2MwjvUX77CzjM0TZGNfxRZfP2LvB-QLN9umw/edit

### CODE REVIEW

File: src/views/ethics/l10n.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+  "ethics.intro1": "At the Scratch Foundation, our mission is to provide young people with digital tools and opportunities to imagine, create, share, and learn. The Scratch platform is a free, safe, playful learning environment that engages all children in thinking creatively, reasoning systematically, and working collaboratively — essential skills for everyone in today's society. We work with educators and families to support children in this playful learning environment.",
+  "ethics.intro2": "This Code of Ethics guides Scratch Foundation employees and affiliates as they conduct research and evaluation in support of the efforts to improve the platform, participant experience, teaching and learning outcomes, and grant-funded work. All research and evaluation work at the Foundation adheres to the following principles.",
+  "ethics.principle1Title": "Principle 1: Protect our participants.",
+  "ethics.researchEmailLinkText": "research@scratchfoundation.org",
```
Comment (apple502j) [inline src/views/ethics/l10n.json:6]: Email should not be translated.

File: src/views/ethics/ethics.jsx [hunk 2]
```diff
@@ -25,6 +25,11 @@ const Ethics = () => (
                         <strong>
                             <FormattedMessage id="ethics.principle1Title" />
                         </strong>
+                    ),
+                    researchEmailLink: (
+                        <a href="mailto:research@scratchfoundation.org">
+                            <FormattedMessage id="ethics.researchEmailLinkText" />
```
Comment (seotts) [inline src/views/ethics/ethics.jsx:31]:
```suggestion
                            research@scratchfoundation.org
```

File: src/views/ethics/l10n.json (hunk 1 above)
Comment (seotts) [inline src/views/ethics/l10n.json:6]:
```suggestion
```

**Review by seotts — CHANGES_REQUESTED** [review]
This looks good! Like apple502j shared, we don't translate emails. I made some suggests for how to include email instead.

**Pull request merged**